bootstrap.php

```diff
@@ -85,9 +85,6 @@
     --as_number <N>     Works with "--as_populate" key only. Number of
                         ActivityStream records for each module record (default 10)
 
-    --as_buffer <N>     Works with "--as_populate" key only. Size of ActivityStream
```

Comment:
I guess we should not remove as_buffer arguments, cause it's a different buffer size and usually it will be bigger that inster_buffer_size